jobs/telnyx-python-developer.html

```diff
@@ -0,0 +1,77 @@
+---
+title: Python Develooer 
+company: Telnyx
+url: https://telnyx.com/company/careers
+location: Dublin, Ireland + Warsaw, Poland +Chicago, USA
```

Comment:
Location can only be one single place. If you have multiple locations you can list them as multiple jobs. Location should describe the specific area where a job is, for example "Bermondsey, London", "Meatpacking District, Manhattan, New York, NY". This allows candidates to find jobs near to them.